perfkitbenchmarker/providers/gcp/gce_virtual_machine.py

```diff
@@ -131,7 +190,11 @@ def _GetOptionDecoderConstructions(cls):
     """
     result = super(CustomMachineTypeSpec, cls)._GetOptionDecoderConstructions()
     result.update({'cpus': (option_decoders.IntDecoder, {'min': 1}),
-                   'memory': (MemoryDecoder, {})})
+                   'memory': (MemoryDecoder, {}),
+                   'gpus': (GceGpuDecoder, {
```

Comment:
Should PKB support this then? It looks like you either need to pass in memory/cpu/gpus OR a named machine type.